[rospy.client][INFO] 2019-06-28 14:55:09,001: init_node, name[/joint_state_publisher], pid[14236]
[xmlrpc][INFO] 2019-06-28 14:55:09,001: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 14:55:09,001: Started XML-RPC server [http://dc5b429f784e:34297/]
[rospy.init][INFO] 2019-06-28 14:55:09,002: ROS Slave URI: [http://dc5b429f784e:34297/]
[rospy.impl.masterslave][INFO] 2019-06-28 14:55:09,002: _ready: http://dc5b429f784e:34297/
[xmlrpc][INFO] 2019-06-28 14:55:09,002: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 14:55:09,003: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 14:55:09,102: registered with master
[rospy.rosout][INFO] 2019-06-28 14:55:09,102: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 14:55:09,105: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 14:55:09,106: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 14:55:09,195: Centering
[rospy.internal][INFO] 2019-06-28 14:55:09,423: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 14:56:06,401: init_node, name[/joint_state_publisher], pid[14424]
[xmlrpc][INFO] 2019-06-28 14:56:06,401: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 14:56:06,401: Started XML-RPC server [http://dc5b429f784e:35427/]
[rospy.init][INFO] 2019-06-28 14:56:06,402: ROS Slave URI: [http://dc5b429f784e:35427/]
[rospy.impl.masterslave][INFO] 2019-06-28 14:56:06,402: _ready: http://dc5b429f784e:35427/
[xmlrpc][INFO] 2019-06-28 14:56:06,403: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 14:56:06,403: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 14:56:06,502: registered with master
[rospy.rosout][INFO] 2019-06-28 14:56:06,503: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 14:56:06,505: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 14:56:06,506: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 14:56:06,588: Centering
[rospy.internal][INFO] 2019-06-28 14:56:06,878: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 14:56:40,941: init_node, name[/joint_state_publisher], pid[14612]
[xmlrpc][INFO] 2019-06-28 14:56:40,942: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 14:56:40,942: Started XML-RPC server [http://dc5b429f784e:41345/]
[rospy.init][INFO] 2019-06-28 14:56:40,943: ROS Slave URI: [http://dc5b429f784e:41345/]
[rospy.impl.masterslave][INFO] 2019-06-28 14:56:40,943: _ready: http://dc5b429f784e:41345/
[xmlrpc][INFO] 2019-06-28 14:56:40,943: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 14:56:40,944: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 14:56:41,043: registered with master
[rospy.rosout][INFO] 2019-06-28 14:56:41,043: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 14:56:41,046: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 14:56:41,048: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 14:56:41,167: Centering
[rospy.internal][INFO] 2019-06-28 14:56:41,482: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 14:58:04,274: init_node, name[/joint_state_publisher], pid[14799]
[xmlrpc][INFO] 2019-06-28 14:58:04,274: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 14:58:04,275: Started XML-RPC server [http://dc5b429f784e:45583/]
[rospy.impl.masterslave][INFO] 2019-06-28 14:58:04,275: _ready: http://dc5b429f784e:45583/
[xmlrpc][INFO] 2019-06-28 14:58:04,276: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2019-06-28 14:58:04,276: ROS Slave URI: [http://dc5b429f784e:45583/]
[rospy.registration][INFO] 2019-06-28 14:58:04,277: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 14:58:04,377: registered with master
[rospy.rosout][INFO] 2019-06-28 14:58:04,378: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 14:58:04,380: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 14:58:04,381: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 14:58:04,463: Centering
[rospy.internal][INFO] 2019-06-28 14:58:04,748: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 14:58:53,729: init_node, name[/joint_state_publisher], pid[14987]
[xmlrpc][INFO] 2019-06-28 14:58:53,730: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 14:58:53,730: Started XML-RPC server [http://dc5b429f784e:41631/]
[rospy.impl.masterslave][INFO] 2019-06-28 14:58:53,730: _ready: http://dc5b429f784e:41631/
[rospy.init][INFO] 2019-06-28 14:58:53,731: ROS Slave URI: [http://dc5b429f784e:41631/]
[xmlrpc][INFO] 2019-06-28 14:58:53,732: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 14:58:53,732: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 14:58:53,831: registered with master
[rospy.rosout][INFO] 2019-06-28 14:58:53,832: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 14:58:53,835: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 14:58:53,836: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 14:58:53,923: Centering
[rospy.internal][INFO] 2019-06-28 14:58:54,211: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 15:00:29,707: init_node, name[/joint_state_publisher], pid[15175]
[xmlrpc][INFO] 2019-06-28 15:00:29,707: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 15:00:29,708: Started XML-RPC server [http://dc5b429f784e:41895/]
[rospy.init][INFO] 2019-06-28 15:00:29,708: ROS Slave URI: [http://dc5b429f784e:41895/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:00:29,708: _ready: http://dc5b429f784e:41895/
[xmlrpc][INFO] 2019-06-28 15:00:29,709: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 15:00:29,710: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 15:00:29,809: registered with master
[rospy.rosout][INFO] 2019-06-28 15:00:29,809: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:00:29,812: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:00:29,813: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 15:00:29,923: Centering
[rospy.internal][INFO] 2019-06-28 15:00:30,208: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 15:01:21,098: init_node, name[/joint_state_publisher], pid[15363]
[xmlrpc][INFO] 2019-06-28 15:01:21,098: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 15:01:21,099: Started XML-RPC server [http://dc5b429f784e:46707/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:01:21,099: _ready: http://dc5b429f784e:46707/
[rospy.init][INFO] 2019-06-28 15:01:21,099: ROS Slave URI: [http://dc5b429f784e:46707/]
[xmlrpc][INFO] 2019-06-28 15:01:21,100: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 15:01:21,101: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 15:01:21,200: registered with master
[rospy.rosout][INFO] 2019-06-28 15:01:21,200: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:01:21,203: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:01:21,205: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 15:01:21,295: Centering
[rospy.internal][INFO] 2019-06-28 15:01:21,637: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 15:02:25,575: init_node, name[/joint_state_publisher], pid[15551]
[xmlrpc][INFO] 2019-06-28 15:02:25,576: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 15:02:25,576: Started XML-RPC server [http://dc5b429f784e:38807/]
[rospy.init][INFO] 2019-06-28 15:02:25,577: ROS Slave URI: [http://dc5b429f784e:38807/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:02:25,577: _ready: http://dc5b429f784e:38807/
[xmlrpc][INFO] 2019-06-28 15:02:25,578: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 15:02:25,578: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 15:02:25,677: registered with master
[rospy.rosout][INFO] 2019-06-28 15:02:25,677: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:02:25,680: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:02:25,682: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 15:02:25,765: Centering
[rospy.internal][INFO] 2019-06-28 15:02:26,040: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 15:04:55,539: init_node, name[/joint_state_publisher], pid[15737]
[xmlrpc][INFO] 2019-06-28 15:04:55,540: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 15:04:55,540: Started XML-RPC server [http://dc5b429f784e:35475/]
[rospy.init][INFO] 2019-06-28 15:04:55,540: ROS Slave URI: [http://dc5b429f784e:35475/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:04:55,541: _ready: http://dc5b429f784e:35475/
[xmlrpc][INFO] 2019-06-28 15:04:55,541: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 15:04:55,542: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 15:04:55,641: registered with master
[rospy.rosout][INFO] 2019-06-28 15:04:55,641: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:04:55,644: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:04:55,647: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 15:04:55,738: Centering
[rospy.client][INFO] 2019-06-28 15:05:33,219: init_node, name[/joint_state_publisher], pid[15901]
[xmlrpc][INFO] 2019-06-28 15:05:33,219: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 15:05:33,220: Started XML-RPC server [http://dc5b429f784e:44035/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:05:33,220: _ready: http://dc5b429f784e:44035/
[rospy.init][INFO] 2019-06-28 15:05:33,220: ROS Slave URI: [http://dc5b429f784e:44035/]
[xmlrpc][INFO] 2019-06-28 15:05:33,221: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 15:05:33,222: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 15:05:33,321: registered with master
[rospy.rosout][INFO] 2019-06-28 15:05:33,321: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:05:33,323: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:05:33,325: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 15:05:33,415: Centering
[rospy.internal][INFO] 2019-06-28 15:05:33,673: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 15:06:35,539: init_node, name[/joint_state_publisher], pid[16089]
[xmlrpc][INFO] 2019-06-28 15:06:35,540: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2019-06-28 15:06:35,540: ROS Slave URI: [http://dc5b429f784e:35637/]
[xmlrpc][INFO] 2019-06-28 15:06:35,540: Started XML-RPC server [http://dc5b429f784e:35637/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:06:35,541: _ready: http://dc5b429f784e:35637/
[xmlrpc][INFO] 2019-06-28 15:06:35,541: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 15:06:35,542: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 15:06:35,641: registered with master
[rospy.rosout][INFO] 2019-06-28 15:06:35,641: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:06:35,643: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:06:35,644: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 15:06:35,729: Centering
[rospy.internal][INFO] 2019-06-28 15:06:35,963: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.client][INFO] 2019-06-28 15:32:41,581: init_node, name[/joint_state_publisher], pid[16280]
[xmlrpc][INFO] 2019-06-28 15:32:41,581: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 15:32:41,582: Started XML-RPC server [http://dc5b429f784e:45443/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:32:41,582: _ready: http://dc5b429f784e:45443/
[rospy.init][INFO] 2019-06-28 15:32:41,582: ROS Slave URI: [http://dc5b429f784e:45443/]
[xmlrpc][INFO] 2019-06-28 15:32:41,583: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 15:32:41,584: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 15:32:41,683: registered with master
[rospy.rosout][INFO] 2019-06-28 15:32:41,684: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:32:41,686: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:32:41,687: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 15:32:41,774: Centering
[rospy.client][INFO] 2019-06-28 15:37:39,183: init_node, name[/joint_state_publisher], pid[16429]
[xmlrpc][INFO] 2019-06-28 15:37:39,183: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 15:37:39,183: Started XML-RPC server [http://dc5b429f784e:44533/]
[rospy.init][INFO] 2019-06-28 15:37:39,184: ROS Slave URI: [http://dc5b429f784e:44533/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:37:39,183: _ready: http://dc5b429f784e:44533/
[xmlrpc][INFO] 2019-06-28 15:37:39,185: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-28 15:37:39,185: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-28 15:37:39,284: registered with master
[rospy.rosout][INFO] 2019-06-28 15:37:39,284: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:37:39,288: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:37:39,289: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2019-06-28 15:37:39,382: Centering
[rospy.internal][INFO] 2019-06-28 15:45:02,325: topic[/joint_states] adding connection to [/rostopic_19061_1561736701947], count 0
[rospy.internal][INFO] 2019-06-28 15:45:05,067: topic[/joint_states] removing connection to /rostopic_19061_1561736701947
[rospy.client][INFO] 2019-06-28 15:45:48,600: init_node, name[/joint_state_publisher], pid[19179]
[xmlrpc][INFO] 2019-06-28 15:45:48,601: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-28 15:45:48,602: Started XML-RPC server [http://dc5b429f784e:35631/]
[rospy.init][INFO] 2019-06-28 15:45:48,601: ROS Slave URI: [http://dc5b429f784e:35631/]
[rospy.impl.masterslave][INFO] 2019-06-28 15:45:48,602: _ready: http://dc5b429f784e:35631/
[rospy.registration][INFO] 2019-06-28 15:45:48,603: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2019-06-28 15:45:48,603: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2019-06-28 15:45:48,702: registered with master
[rospy.rosout][INFO] 2019-06-28 15:45:48,703: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-28 15:45:48,705: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-28 15:45:48,707: initializing /clock core topic
[rospy.simtime][INFO] 2019-06-28 15:45:48,711: connected to core topic /clock
[rosout][INFO] 2019-06-28 15:45:48,824: Centering
[rospy.internal][INFO] 2019-06-28 15:45:56,739: topic[/clock] adding connection to [http://dc5b429f784e:38065/], count 0
[rospy.internal][ERROR] 2019-06-28 15:48:37,260: Unable to initiate TCP/IP socket to dc5b429f784e:40847 (http://dc5b429f784e:38065/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 40847 to [172.17.0.2:41864 on socket 17]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-28 15:48:38,262: Unable to initiate TCP/IP socket to dc5b429f784e:40847 (http://dc5b429f784e:38065/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 40847 to [172.17.0.2:41926 on socket 17]] [/joint_state_publisher].

[rospy.internal][WARNING] 2019-06-28 15:48:40,266: Unknown error initiating TCP/IP socket to dc5b429f784e:40847 (http://dc5b429f784e:38065/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][INFO] 2019-06-28 15:48:40,266: topic[/clock] removing connection to http://dc5b429f784e:38065/
[rospy.internal][INFO] 2019-06-28 15:48:48,704: topic[/clock] adding connection to [http://dc5b429f784e:44839/], count 0
[rospy.internal][ERROR] 2019-06-28 15:51:07,646: Unable to initiate TCP/IP socket to dc5b429f784e:49189 (http://dc5b429f784e:44839/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 49189 to [172.17.0.2:56250 on socket 17]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-28 15:51:08,648: Unable to initiate TCP/IP socket to dc5b429f784e:49189 (http://dc5b429f784e:44839/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 49189 to [172.17.0.2:56322 on socket 17]] [/joint_state_publisher].

[rospy.internal][WARNING] 2019-06-28 15:51:10,648: Unknown error initiating TCP/IP socket to dc5b429f784e:49189 (http://dc5b429f784e:44839/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][INFO] 2019-06-28 15:51:10,649: topic[/clock] removing connection to http://dc5b429f784e:44839/
[rospy.internal][INFO] 2019-06-28 15:51:13,151: topic[/clock] adding connection to [http://dc5b429f784e:34161/], count 0
[rospy.internal][ERROR] 2019-06-28 15:51:47,026: Unable to initiate TCP/IP socket to dc5b429f784e:39727 (http://dc5b429f784e:34161/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 39727 to [172.17.0.2:41932 on socket 17]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-28 15:51:48,027: Unable to initiate TCP/IP socket to dc5b429f784e:39727 (http://dc5b429f784e:34161/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 39727 to [172.17.0.2:41994 on socket 17]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-28 15:51:50,030: Unable to initiate TCP/IP socket to dc5b429f784e:39727 (http://dc5b429f784e:34161/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 39727 to [172.17.0.2:41996 on socket 17]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-28 15:51:54,035: Unable to initiate TCP/IP socket to dc5b429f784e:39727 (http://dc5b429f784e:34161/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 39727 to [172.17.0.2:41998 on socket 17]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-28 15:52:02,037: Unable to initiate TCP/IP socket to dc5b429f784e:39727 (http://dc5b429f784e:34161/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 39727 to [172.17.0.2:42000 on socket 17]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-28 15:52:18,041: Unable to initiate TCP/IP socket to dc5b429f784e:39727 (http://dc5b429f784e:34161/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 39727 to [172.17.0.2:42002 on socket 17]] [/joint_state_publisher].

